### DISCUSSION TITLE
Introduction to a new installation : video in French only

### DISCUSSION THREAD

**Discussion by guillaumarchais** [mcguirepr89/BirdNET-Pi#1123] [Show and tell]
Hello, I have stopped my home station last November and I wanted to get back on track for the coming season 2024. It has taken me a while to get it right as the new installation did not completely work first time when updating the same RPi4b device. So after browsing a lot on the discussions panel to get answers for my tech issues, I have decided to compile the essential bits and make an introduction video in French to help promoting this piece of work for the greater good.
Here it is https://youtu.be/3ZqwmGAhz9o?si=XOWwzAYOH5oq11mx
Non professionnal youtuber... just amateur stuff.

**Comment by blogdemoi** [comment]
As a French, I say "Merci"!

**Reply by guillaumarchais** [reply]
Merci ! I have taken a quick look at your website https://birdnetpi.arnauld.org
I am looking into making it publi mine too but not using the DDNS solution suggested. I have a free.fr page perso website I could use as http://monsite.free.fr/birdnet. I'll be glad if you could explain me how you made yours public.

**Reply by blogdemoi** [reply]
I have a server (a Pi 3b) at home running Yunohost https://yunohost.org/fr/whatsyunohost on which I have a blog, Nextcloud, pictures, accessible from outside (I have a static IP address). 

I installed Birdnet-Pi on another Pi (actually a Le Potato card), which gives me a local Ip. In Yunohost I installed an app, Redirect https://github.com/YunoHost-Apps/redirect_ynh , which makes it possible to have my Birdnet-Pi accessible to the outside word.

I think this way is too much work if you just need Birdnet-Pi...

**Reply by guillaumarchais** [reply]
OK thank you for your explanation. I have found out that free pages perso does not allow a DNS - A record with another IP.
I ended up using NO-IP again for now http://gmarchaisbirdnet1.sytes.net

**Comment by blogdemoi** [comment]
> I ended up using NO-IP again for now http://gmarchaisbirdnet1.sytes.net

Listening now to your "Chouette Hulotte". I guess it is more common to hear such birds where you live than my location near Paris.
